src/mods/timer.cpp

```diff
@@ -52,9 +165,40 @@ void disp() {
         }
     }
 
-    if (pref::get(pref::BoolPref::RtaPauseTimer) && !freecam::should_hide_hud()) {
-        timerdisp::draw_timer(s_rta_timer, "RTA:", 1, draw::WHITE, true);
-        timerdisp::draw_timer(s_pause_timer, "PAU:", 2, draw::WHITE, true);
+    u32 row = 1;
+
+    if (pref::get(pref::BoolPref::TimerShowRTA) && !freecam::should_hide_hud()) {
+        timerdisp::draw_timer(s_rta_timer, "RTA:", row++, draw::WHITE, true);
+    }
+
+    if (pref::get(pref::BoolPref::TimerShowPause) && !freecam::should_hide_hud()) {
+        timerdisp::draw_timer(s_pause_timer, "PAU:", row++, draw::WHITE, true);
+    }
+
+    switch (mkb::sub_mode) {
+        case mkb::SMD_GAME_GOAL_INIT:
+        case mkb::SMD_GAME_GOAL_MAIN:
+        case mkb::SMD_GAME_GOAL_REPLAY_INIT:
+        case mkb::SMD_GAME_GOAL_REPLAY_MAIN:
+            break;
+        default:
+            return;
+    }
+
+    if (pref::get(pref::BoolPref::TimerShowSubtick) && !freecam::should_hide_hud()) {
+        timerdisp::draw_subtick_timer(mkb::mode_info.stage_time_frames_remaining, "SUB:", row++,
+                                      draw::WHITE, true, s_framesave, false);
+    }
+
+    if (pref::get(pref::BoolPref::TimerShowUnrounded) && !freecam::should_hide_hud()) {
+        timerdisp::draw_subtick_timer(mkb::mode_info.stage_time_frames_remaining, "CUR:", row++,
+                                      draw::WHITE, true, 0, false);
+        timerdisp::draw_subtick_timer(mkb::mode_info.stage_time_frames_remaining + 1, "NXT:", row++,
+                                      draw::WHITE, true, 0, false);
+    }
```

Comment:
I wish we could make it obvious somehow that "unrounded" refers to the normal framecount time but displayed more accurately - "unrounded" kind of sounds like it might be the subframe time

Reply:
Yeah I wasn't too sure on the wording either, I'm fine waiting to see if we can come up with anything better. I think some help text would also help a lot with explaining all the different timers and their uses. Not sure if it would be better to incorporate that in the timer menu itself or in a separate help page.

Reply:
I'm cool with taking this as-is for now, and if you think of how you want to show help information we can add that later.